app/src/main/java/chat/rocket/android/main/ui/MainActivity.kt

```diff
@@ -111,6 +111,12 @@ class MainActivity : AppCompatActivity(), MainView, HasActivityInjector, HasSupp
 
             expanded = !expanded
         }
+
+        header.image_avatar.setOnClickListener {
+            view_navigation.menu.getItem(1).isChecked = true
```

Comment:
Please user `view_navigation.menu.findItem(R.id.action_profile)` instead of getting by the index since it may change anytime.